src/Generators/NovaTestGenerator.php

```diff
@@ -48,6 +48,7 @@ public function generateTests(): void
             'entity' => $this->model,
             'entities' => $this->getPluralName($this->model),
             'lower_entity' => Str::snake($this->model),
+            'lower_first_entity' => Str::lcfirst($this->model),
```

Comment:
it would be better to rename these keys

```suggestion
            'snake_entity' => Str::snake($this->model),
            'dromedary_entity' => Str::lcfirst($this->model),
```